src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1358,8 +1358,7 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 		}
 		// If this is a automation-length-edit press...
 		//needed for Automation
```

Comment:
Removed time check between presses, which allows you to press two pads consecutively and it will select those two pads as a long press